BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -715,9 +730,16 @@ private void ConvertTriggerPush(Entity trigger)
 
 		private void ConvertTriggerTeleport(Entity trigger)
 		{
+			var spawnflags = (Q3TriggerTeleportFlags)trigger.Spawnflags;
```

Comment:
Extract this code into a method and reuse it on line 708. It should be called something like `SetTriggerTeleportMode`

Reply:
It doesn't work for that since the trigger_teleport flags are different to the target_teleporter flags. 

Flag value of 2 is keepspeed on trigger_teleport, flag value of 1 is keepspeed on target_teleporter.

Reply:
You are right. I misread the spawnflags and thought they were using the same values.